### PR TITLE
feat: add permissions copy service

### PR DESCRIPTION
**Pull request by gabrielfs7**
Ticket https://oat-sa.atlassian.net/browse/ADF-1031

# Goal

To copy DacSimple ACL permissions while copying resources

## Require PRs
* https://github.com/oat-sa/tao-core/pull/3533

## Tasks
- [x] Update `oat-sa/tao-core` dependency
- [x] Add `oat-sa/tao-items` dependency

### CODE REVIEW

File: model/DataBaseAccess.php [hunk 1]
```diff
@@ -348,7 +348,7 @@ public function removeAllPermissions($resourceIds)
 
         $this->getPersistence()->exec($query, $resourceIds);
         foreach ($resourceIds as $resourceId) {
-            $this->getEventManager()->trigger(new DacRemovedEvent('-', $resourceId, '-'));
+            $this->getEventManager()->trigger(new DacRemovedEvent('-', $resourceId, ['-']));
```
Comment (gabrielfs7) [inline model/DataBaseAccess.php:351]: Wow, it was invalid for 2 years 😲 

Thanks 💪

File: composer.json [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
-    "minimum-stability" : "dev",
+    "minimum-stability": "dev",
     "require": {
         "php": ">=7.1",
         "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
-        "oat-sa/generis" : ">=14.0.0",
-        "oat-sa/tao-core" : ">=48.0.0",
-        "oat-sa/extension-tao-backoffice" : ">=6.0.0"
+        "oat-sa/generis": ">=14.0.0",
+        "oat-sa/tao-core": "dev-feature/adf-1031/copy-dac-simple-permissions as 50.13.0",
```
Comment (gabrielfs7) [inline composer.json:26]: - [ ] Update branch before merge

**Review by shpran — APPROVED** [review]
IDK can I approve the PR I am in, but approved 😂

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Comment by AntonNeviarouski** [comment]
QA verified

File: composer.json (hunk 2 above)
Comment (shpran) [inline composer.json:26]:
```suggestion
        "oat-sa/tao-core": ">=50.15.0",
```

**Pull request merged**